### DISCUSSION TITLE
Обрыв операции

### DISCUSSION THREAD

**Discussion by F1ame3103** [MrZillaGold/VK2Discord#548] [Q&A] [answered]
Здравствуйте, делал все по инструкции VK2Discord вроде как запускается но при этом ничего не происходит ошибок нету,
пробовал разные версии одно и то же, вот скрин

![image](https://user-images.githubusercontent.com/108095687/175391965-771c3647-652b-4a5d-9d20-be9ac3b21383.png)

**Comment by egorprnn** [comment, marked as answer]
Похоже вы находитесь в стране в которой заблокирован VK, используйте VPN
